### DISCUSSION TITLE
Please add Ezra to Winget on Windows

### DISCUSSION THREAD

**Discussion by southpawiam** [ezra-bible-app/ezra-bible-app#946] [Ideas]
I had a catastrophic fail on Windows resulting in a fully reset operating system recently.  I looked for some "newer" option for installations and there was an option called winget in powershell 7, and likely earlier(You can disable powershell 2, uninstall the microsoft store powershell 5 and just use powershell 7 and winget - https://winstall.app/ ).  It's amazing.  I installed all the software I could through winget, and I have found software dependencies update through winget easily(including dependencies from software installed by Steam).    

However, I also found that very little Bible software is using winget which is why I'm requesting this.  There's BPBible, which requires additions separately downloaded through a web browser and Logos 9.  Other than than, there's nothing Bible there.  There's not even Bible memory apps.

**Comment by tobias-klein** [comment]
Interesting! There is also Chocolatey, which I have been using as a package manager on Windows. It works quite well.

https://community.chocolatey.org/packages

They seem to have more packages than Winget. One open source Bible software they have in the repo is BibleTime.

Before doing that we may need to look into signing the binaries so that they are securely identified. That's already done on macOS, but not yet on Windows and if you currently execute the Ezra installer you usually get a big warning message about an unknown piece of software downloaded from the internet. I would guess that that scares the average user.

**Reply by southpawiam** [reply]
I've heard of Chocolatey.  Winget was built into Powershell 7, there was nothing to install, so I figured I'd see if it worked - and it did.  Though, looking at 9,000 to 5,000 , maybe I should have used Chocolatey.

I'm sure you're right on signing binaries.  There were a few installs in winget that did their business without ever giving me a dialogue box, but most of them just liked giving me a 'do you want to install this on your computer gui' on install for some reason or another.  There were also a few others that simply asked me in the shell y or n questions too.  It looks like there's a bit of play-testing to it to be sure the user experience is what you assume it will be.

**Reply by southpawiam** [reply]
Perhaps it is needless to mention, but Microsoft Windows SHA2 is still lingering in minds even after Windows 11 23H2 supposedly changed it to SHA3.  SHA2 ( https://en.wikipedia.org/wiki/SHA-2 ) has a safe 384, and pretty much everything else is not great.  So, stuff signed with SHA384 feels better on Windows.  I just figured I'd mention that... since you mentioned signing.
